src/Dapr.Workflow/WorkflowRuntimeOptions.cs

```diff
@@ -65,10 +65,10 @@ public sealed class WorkflowRuntimeOptions
         }
 
         /// <summary>
-        /// Method to add workflow to the registry.
+        /// Method to add activities to the registry.
         /// </summary>
-        /// <param name="registry">The registry we will add workfows to</param>
-        internal void AddWorkflowsToRegistry(DurableTaskRegistry registry)
+        /// <param name="registry">The registry we will add activities to</param>
+        internal void AddActivitiesToRegistry(DurableTaskRegistry registry)
```

Comment:
It seems to me that the code in this method works for both workflows and activities. If we want to be really specific, we'd probably want to name this `AddWorkflowsAndActivitiesToRegistry`.

Reply:
But it doesn't really add/register any workflow with the registry, does it?

Reply:
I'm in favor of being more specific here, though if anyone feels super strongly we can leave it as is.

Reply:
Done in 509656b35a9282d4740a781c1800005b96b36262

Reply:
Resolving.